qemu/tests/cfg/openflow_acl_test.cfg

```diff
@@ -76,7 +76,7 @@
             ftp:
                 setup_targets = localhost
             website:
-                access_cmd_linux = lynx --dump www.redhat.com
+                access_cmd_linux = curl www.redhat.com
```

Comment:
when sebserver doesn't work, curl command will be in a wait state. so I suggest add this option --connect-timeout.  eg.  curl -l http://10.10.10.10 --connect-timeout 10.